### PR TITLE
[RTAPI-9] - Updated authenciation to use cookies instead of jwt token…

### PR DESCRIPTION
**Pull request by cpb2948**
…s. JWT code still exists

### CODE REVIEW

**Comment by cpb2948** [comment]
@dadleyy 

Update authentication to use cookies which makes sense. the code for JWT tokens still exists, i want to find a way to implement both for the application. For now we will still with cookies.

File: app/Http/routes.php [hunk 1]
```diff
@@ -16,17 +16,19 @@
 Route::post('/register', 'RegistrationController@Register');
 
 //Auth Routes
-Route::get('/auth', 'AuthenticationController@Authenticate');
+Route::get('/autht', 'AuthenticationController@AuthenticateToken');
+Route::get('/authc', 'AuthenticationController@AuthenticateCookie');
```
Comment (dadleyy) [inline app/Http/routes.php:20]: why are you doing `GET` routes for this? shouldnt it be `POST`?

File: config/session.php [hunk 2]
```diff
@@ -16,7 +16,7 @@
     |
     */
 
-    'driver' => env('SESSION_DRIVER', 'array'),
+    'driver' => env('SESSION_DRIVER', 'file'),
```
Comment (dadleyy) [inline config/session.php:19]: not that we have to do it now, but we should definitely see if we can get [redis](http://redis.io/) to be our session driver

**Comment by cpb2948** [comment]
Yes they should be posts, my bad.

And yes down the road we should use redis

**Pull request merged**